src/main/webapp/resources/js/pages/dashboard/components/RecentActivity.jsx

```diff
@@ -0,0 +1,59 @@
+import { Button, Space, Typography } from "antd";
+import React from "react";
+import { grey1 } from "../../../styles/colors";
+import { SPACE_MD } from "../../../styles/spacing";
+import { RecentActivityAllProjects } from "./RecentActivityAllProjects";
+import { RecentActivityUserProjects } from "./RecentActivityUserProjects";
+
+const isAdmin = window.TL._USER.systemRole === "ROLE_ADMIN";
+
+/**
+ * Component to display recent activity
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function RecentActivity() {
+  const [allProjectsVisible, setAllProjectsVisible] = React.useState(false);
+  const [userProjectsVisible, setUserProjectsVisible] = React.useState(true);
+
+  return (
+    <Space
```

Comment:
Space does work good here, lately I have been findign that using there Grid system actually works slightly better.  Wrap each in a `Col` tag and Wrap everything in a `<Row gutter={[16,16]}>` seems to handle it very well.

Reply:
Thanks! Yeah it works great. Updated in 0042a0ad2e7a59b91ddeb0ff9efe5adaa46c6c25